site/content/docs/developing/addons/environment.en.md

```diff
@@ -0,0 +1,274 @@
+To do so, you should first export the value from the environment addon using the `Outputs` section.
+
+???+ note "Example: Export values from environment addons"
+    ```yaml
+    Outputs:
+        MyTableARN:
+            Value: !GetAtt ServiceTable.Arn
+            Export:
+                Name: !Sub ${App}-${Env}-MyTableARN  # This value can be consumed by a workload manifest or a workload addon.
+        MyTableName:
+            Value: !Ref ServiceTable
+            Export:
+                Name: !Sub ${App}-${Env}-MyTableName
+    ```
+
+
+It is important that you add the `Export` block. 
+Otherwise, your workload stack or your workload addons won't be able to access the value.
+You will use `Export.Name` to reference the value from your workload-level resources.
+
+???- hint "Consideration: Namespace your `Export.Name`"
+    You can specify any name you like for `Export.Name`.
+    That is, it doesn't have to be prefixed with `!Sub ${App}-${Env}`; it can simply be `MyTableName`.
+
+    However, within an AWS region, the `Export.Name` must be unique. 
+    That is, you can't have two exports named `MyTableName` in `us-east-1`.
+    
+    Therefore, we recommend you to namespace your exports with `${App}` and `${Env}` to decrease the chance of name collision. 
+    In addition, this makes it clear which application and environment the value is managed under.
+    
+    With the namespace, for example, say your application's name is `"my-app"`,
+    and you deployed the addons with environment `test`, then the final export name would be rendered to  `my-app-test-MyTableName`.
+
+
+##### Referencing from a workload addon
+
+In your workload addons, you can reference a value from your environment addons, as long as that value is exported.
+To do so, use [`Fn::ImportValue`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-importvalue.html)
+function with that value's export name to import it from an environment addon.
+
+???- note "Example: An IAM policy to access an environment-level DynamoDB table"
+    ```yaml
+    Parameters:
+      App:
+        Type: String
+        Description: Your application's name.
+      Env:
+        Type: String
+        Description: The environment name your service, job, or workflow is being deployed to.
+      Name:
+        Type: String
+        Description: The name of the service, job, or workflow being deployed.
+    Resources:
+      MyTableAccessPolicy:
+        Type: AWS::IAM::ManagedPolicy
+        Properties:
+          Description: Grants CRUD access to the Dynamo DB table
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Sid: DDBActions
+                Effect: Allow
+                Action:
+                  - dynamodb:* # NOTE: Scope down the permissions in your real application. This is done so that the blog isn't too long!
```

Comment:
```suggestion
                  - dynamodb:* # NOTE: Scope down the permissions in your real application. This is done so that this example isn't too long!
```